jstz_cli/src/config.rs

```diff
@@ -8,12 +9,48 @@ use std::{
 use anyhow::{anyhow, Result};
 use serde::{Deserialize, Serialize};
 
+use crate::accounts::account::Account;
+
 fn home() -> PathBuf {
     dirs::home_dir()
         .expect("Failed to get home directory")
         .join(".jstz")
 }
 
+// Represents a collection of accounts
+#[derive(Serialize, Deserialize, Debug, Default, Clone)]
+pub struct AccountConfig {
+    accounts: HashMap<String, Account>,
+}
+
+impl AccountConfig {
+    /*
+    pub fn new() -> Self {
+        AccountList {
+            accounts: HashMap::new(),
+        }
+    }
+    */
```

Comment:
Please avoid committing possibly dead code in comments, these kinds of comments often bit rot quickly. 

If you have local changes that you'd like to perhaps use in the future just commit sections of the file and stash the changes you'd like to keep for later